### ISSUE TITLE
flaws of ScrollView sticky headers on Android

### ISSUE THREAD

**Issue by nihgwu** [facebook/react-native#9929]
Thanks for @janicduplessis 's great work #9456 ,now we can stick the section headers on Android as on iOS, and I can remove my js workaround finally, I've test it and works really awesome with little flaws: 

1. The sticky headers will bounce if `LayoutAnimation` is configured, just try the `ListViewPagingExample` you can see that sticky headers will be affected by `LayoutAnimation`
2. If `renderHeader` is set to `ListView`, and the header is a `View` component with no children, then the first item of the section rows will be stuck instead of the section header.
```js
//good
renderHeader= () => {
  <View style={styles.header} ><View /></View>;
}
//bad
renderHeader= () => {
  <View style={styles.header} />;
}
```
but if you use `Text` or `TouchableOpacity` instead of `View` with no children, it works accurately too

**Comment by janicduplessis** [comment]
Thanks a lot for testing this, gonna have a look at these issues.

**Comment by nihgwu** [comment]
@janicduplessis I've made a PR #9930 to add the `ListViewPagingExample` to the example list 😄

**Comment by nihgwu** [comment]
close as being resolved in #9957
